src/Compute/Compute/Extension/AEM/SetAzureRmVMAEMExtension.cs

```diff
@@ -125,6 +125,38 @@ public class SetAzureRmVMAEMExtension : VirtualMachineExtensionBaseCmdlet
                 HelpMessage = "Enable debug mode for the VM Extension.")]
         public SwitchParameter DebugExtension { get; set; }
 
+        //[Parameter(
+        //    Mandatory = false,
+        //    Position = 9,
+        //    ParameterSetName = "NewExtension",
+        //    ValueFromPipelineByPropertyName = false,
+        //    HelpMessage = "Configures client_id.")]
+        //public string ClientId { get; set; }
+
+        [Parameter(
+            Mandatory = false,
+            Position = 9,
```

Comment:
large-digit positional parameters are generally discouraged because no one can remember the order when there are so many positional parameters.